docs/commands/AggregateIntervals.md

```diff
@@ -0,0 +1,103 @@
+# `aggregateintervals`
+
+This command allows you to compute statistics for a set of genomic intervals.
+
+#### Command Line Arguments
+
+Argument | Shortcut | Default | Description
+:-:  | :-: |:-: | ---
+`--input <intervals path>` | `-i` | **Required** | Path to interval list file
+`--output <export path>` | `-o` | **Required** | Path for output text file
+`--condition <expr>` | `-c` | **Required** | Export expression (see below)
+
+____
+
+## Designating output with an expression
+An export expression designates a list of computations to perform, and what these columns are named.  These expressions should take the form `COL_NAME_1 = <expression>, COL_NAME_2 = <expression>, ...`.  See the examples for ideas.
+
+**Namespace in the `aggregateintervals` condition:**
+
+Identifier | Description
+:-: | ---
+`interval.contig` | chromosome of interval
+`interval.start` | start position of interval
+`interval.end` | end position of interval
```

Comment:
Should we just expose `start` and `end` as Loci, now that we have them?  That's also how we represent them internally now.

Also, remark that start and end are 1-based, right-inclusive-left-exclusive, following the internal conventions, not following the interval list file conventions.